src/Symfony/Component/Security/Http/Tests/Fixtures/IsGrantedAttributeMethodsWithCallableController.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Security\Http\Tests\Fixtures;
+
+use Symfony\Component\Security\Core\Authentication\Token\TokenInterface;
+use Symfony\Component\Security\Http\Attribute\IsGranted;
+
+class IsGrantedAttributeMethodsWithCallableController
+{
+    public function noAttribute()
+    {
+    }
+
+    #[IsGranted(static function ($token, $subject, $accessDecisionManager, $trustResolver) {
+        return $accessDecisionManager->decide($token, ['ROLE_ADMIN']);
+    })]
+    public function admin()
+    {
+    }
+
+    #[IsGranted(static function ($token, $subject, $accessDecisionManager, $trustResolver) {
+        return $accessDecisionManager->decide($token, ['ROLE_ADMIN']);
+    }, subject: 'arg2Name')]
+    public function withSubject($arg1Name, $arg2Name)
+    {
+    }
+
+    #[IsGranted(static function ($token, $subject, $accessDecisionManager, $trustResolver) {
+        return $accessDecisionManager->decide($token, ['ROLE_ADMIN']);
+    }, subject: ['arg1Name', 'arg2Name'])]
+    public function withSubjectArray($arg1Name, $arg2Name)
+    {
+    }
+
+    #[IsGranted(static function ($token, $subject, $accessDecisionManager, $trustResolver) {
+        return $accessDecisionManager->decide($token, ['ROLE_ADMIN']);
+    }, subject: 'non_existent')]
+    public function withMissingSubject()
+    {
+    }
+
+    #[IsGranted(static function ($token, $subject, $accessDecisionManager, $trustResolver) {
+        return $accessDecisionManager->decide($token, ['ROLE_ADMIN']);
+    }, message: 'Not found', statusCode: 404)]
+    public function notFound()
+    {
+    }
+
+    #[IsGranted(static function ($token, $subject, $accessDecisionManager, $trustResolver) {
+        return $accessDecisionManager->decide($token, ['ROLE_ADMIN']);
+    }, message: 'Exception Code Http', statusCode: 404, exceptionCode: 10010)]
+    public function exceptionCodeInHttpException()
+    {
+    }
+
+    #[IsGranted(static function ($token, $subject, $accessDecisionManager, $trustResolver) {
+        return $accessDecisionManager->decide($token, ['ROLE_ADMIN']);
+    }, message: 'Exception Code Access Denied', exceptionCode: 10010)]
+    public function exceptionCodeInAccessDeniedException()
+    {
+    }
+
+    #[IsGranted(
+        static function (TokenInterface $token, $subject) {
+            return $token->getUser() === $subject;
+        },
+        subject: static function (array $args) {
+            return $args['post'];
+        }
+    )]
+    public function withCallableAsSubject($post)
+    {
+    }
+
+    #[IsGranted(static function ($token, $subject) {
```

Comment:
is that legit? should it be
```suggestion
    #[IsGranted(static function ($token, $subject, ...$vars) {
```

Reply:
That's right, we may add integration tests to check this because unit tests won't be able to catch this fixture error easily